configure

```diff
@@ -1033,6 +1033,7 @@ def glob_to_var(dir_base, dir_sub, patch_dir):
   return list
 
 def configure_intl(o):
+  config_vars = o['variables']
```

Comment:
This introduces a lot of churn for no good reason.  Please don't do that, no reviewer likes a noisy diff.